P11: 0.01
P22: 0.01
P33: 0.01
P44: 0.01
P55: 0.01 
P66: 0.01
P77: 0.01

Ri11: 0.0000000005
Ri22: 0.005
Rw11: 0.1
Rw22: 0.00001

Q11: 10
Q22: 1